cli/cmd/capture/create.go

```diff
@@ -174,10 +173,9 @@ func deleteSecret(kubeClient kubernetes.Interface, secretName *string) error {
 }
 
 func createCaptureF(kubeClient kubernetes.Interface) (*retinav1alpha1.Capture, error) {
-	captureName := fmt.Sprintf("retina-capture-%s", utilrand.String(5))
```

Comment:
Should we keep auto-generation functionality?

Reply:
You have to use the name for `download` and `delete` so I think the UX is better if it also has to be provided during the `create`

Reply:
Makes sense. Also, if you're forced to set the name, perhaps it'll be easier to identify which capture is which in a list of captures